src/storage/sql/migrations/2020-07-25-065258_create_snippets/up.sql

```diff
@@ -0,0 +1,58 @@
+CREATE TABLE snippets (
+    -- an internal autoincrementing identifier used in foreign keys.
+    -- Normally not visible publicly, except for the case when it is
+    -- used for looking legacy snippets up by id
+    id SERIAL PRIMARY KEY,
+    -- a short unique snippet identifier visible to users
+    slug TEXT NOT NULL,
+
+    title TEXT,
+    syntax TEXT,
+    created_at TIMESTAMP WITH TIME ZONE DEFAULT NOW() NOT NULL,
+    updated_at TIMESTAMP WITH TIME ZONE,
```

Comment:
*(nitpicking)* Let's add `DEFAULT NOW() NOT NULL` here too. I feel like it might simplify some queries such as if we want to show some activity feed in the future.

Reply:
Agreed. Done.